new-bot/src/bot/features/download-video.ts

```
@@ -76,7 +84,7 @@ async function resolveInstagramShorthandUrl(url: string): Promise<string | null>
   }
 }
 
-async function processVideoUrl(url: string, ctx: Context): 
+async function processVideoUrl(url: string, ctx: Context, isVideoRequired: boolean = true): 
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)